### ISSUE TITLE
Status page

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#247]
* Separate out Network and Filtering method into separate columns. (Network, Filter, Filter status)

* Use Filter status to state: on / off

* Add new states to status: "testing" for new probes not yet online; "retired" for those no longer in use

* Use tool tip ( i ) to link to information about each filter level as per results pages

* Add column 'Supplied by' (eg, AAISP, ORG, user) (ask users if they want to be credited, otherwise say 'user')

* Add column: total number of tests run

* Put active probes first 

* Have option to show retired probes

(changes following on from #142 which I've closed)

**Comment by JimKillock** [comment]
(amended to include  "Add column: total number of tests run")

**Comment by dantheta** [comment]
All done apart from Filter status on/off: not quite sure if the information we want to convey with that column is already provided by filter level.  Is it to say that filtering is available but disabled (TalkTalk,Plusnet), or is it to say that filtering is not enabled on the ISP at all (AAISP, Uno), or both?

**Comment by JimKillock** [comment]
On filter status, I'd envisaged that to mean "filter is currently working as expected" or "filter is not working as expected", given the problems we've had with TT and Plusnet (filters are enabled but not actually working).  So status could be:

"OK: filter on"; "Error: filter off"

Where there is no filter, the column could indicate an error (filter detected) when a filter accidentally becomes enabled. In these cases:

"Error: filter on" or "OK: unfiltered"

One cosmetic change: can the tests run column use commas eg 21,123,456 ?

**Comment by dantheta** [comment]
OK, that's cool.

**Comment by dantheta** [comment]
Commas are done!

**Comment by dantheta** [comment]
All done.  The filter enabled/disabled doesn't update automatically at the moment, because the probes send no response on a line which should be filtered but isn't.  We can open an issue for probes to feed that back to the server.

**Comment by JimKillock** [comment]
Great work. Yes, please do add a separate issue for the probes to return the filter down result. Or indeed any (unexpected) filter up result.

**Comment by JimKillock** [comment]
Can we add the user probes that are either in testing mode, or are currently working but not listed, eg for KCom and Virgin / unfiltered and so on?

From the copyright blocks it looks like we may also need some users with unfiltered Sky lines etc to check the presentation of some sites which are blocked by filters and also by court orders, as the filters seem to take precedence on some ISPs.

**Comment by JimKillock** [comment]
Query re: Zen; we sent a probe to someone for a Zen line, have we mis-listed it as Gigaclear (we have two Gigaclear)?

**Comment by dantheta** [comment]
The Zen one is listed.  There are two probes for gigaclear, on different subnets on their network.

**Comment by JimKillock** [comment]
Thanks Dan. I think there is a Virgin clear line missing tho?

**Comment by JimKillock** [comment]
This can be closed as we've done most of it, moving the part about filter status into the inbox
